src/highdicom/sr/templates.py

```diff
@@ -482,6 +486,168 @@ def _contains_image_items(
+        '1.2.840.10008.5.1.4.1.1.88.34': codes.cid32.StructuredReportDocument,
+        '1.2.840.10008.5.1.4.1.1.88.35': codes.cid32.StructuredReportDocument,
+        '1.2.840.10008.5.1.4.1.1.88.50': codes.cid32.StructuredReportDocument,
+        '1.2.840.10008.5.1.4.1.1.88.59': codes.cid32.KeyObjectSelection,
+        '1.2.840.10008.5.1.4.1.1.88.65': codes.cid32.StructuredReportDocument,
+        '1.2.840.10008.5.1.4.1.1.88.67': codes.cid32.StructuredReportDocument,
+        '1.2.840.10008.5.1.4.1.1.88.68': codes.cid32.StructuredReportDocument,
+        '1.2.840.10008.5.1.4.1.1.88.70': codes.cid32.StructuredReportDocument,
+        '1.2.840.10008.5.1.4.1.1.88.71': codes.cid32.StructuredReportDocument,
+        '1.2.840.10008.5.1.4.1.1.88.72': codes.cid32.StructuredReportDocument,
+        '1.2.840.10008.5.1.4.1.1.88.73': codes.cid32.StructuredReportDocument,
+        '1.2.840.10008.5.1.4.1.1.88.74': codes.cid32.StructuredReportDocument,
+        '1.2.840.10008.5.1.4.1.1.88.75': codes.cid32.StructuredReportDocument,
+        '1.2.840.10008.5.1.4.1.1.88.76': codes.cid32.StructuredReportDocument,
+        '1.2.840.10008.5.1.4.1.1.90.1': codes.cid32.ContentAssessmentResult,
+        '1.2.840.10008.5.1.4.1.1.128': codes.cid29.PositronEmissionTomography,
+        '1.2.840.10008.5.1.4.1.1.130': codes.cid29.PositronEmissionTomography,
+        '1.2.840.10008.5.1.4.1.1.128.1': codes.cid29.PositronEmissionTomography,
+        '1.2.840.10008.5.1.4.1.1.200.2': codes.cid32.CTProtocol,
+        '1.2.840.10008.5.1.4.1.1.481.1': codes.cid29.RTImage,
+        '1.2.840.10008.5.1.4.1.1.481.2': codes.cid32.RTDose,
+        '1.2.840.10008.5.1.4.1.1.481.3': codes.cid32.RTStructureSet,
+        '1.2.840.10008.5.1.4.1.1.481.4': codes.cid32.RTTreatmentRecord,
+        '1.2.840.10008.5.1.4.1.1.481.5': codes.cid32.RTPlan,
+        '1.2.840.10008.5.1.4.1.1.481.6': codes.cid32.RTTreatmentRecord,
+        '1.2.840.10008.5.1.4.1.1.481.7': codes.cid32.RTTreatmentRecord,
+        '1.2.840.10008.5.1.4.1.1.481.8': codes.cid32.RTPlan,
+        '1.2.840.10008.5.1.4.1.1.481.9': codes.cid32.RTTreatmentRecord,
+    }
+    try:
+        return sopclass_to_modality_map[sop_class_uid]
+    except KeyError:
+        raise ValueError(
+            'SOP Class UID does not identify a SOP Class '
+            'for storage of an image information entity.'
+        )
+
+
+def _is_image(dataset: Dataset) -> bool:
+    """Check whether data set represents an image.
+
+    Parameters
+    ----------
+    dataset: pydicom.dataset.Dataset
+        Dataset
+
+    Returns
+    -------
+    bool
+        ``True`` if `dataset` is an image, ``False`` otherwise
+
+    """
+    if all(key in dataset for key in ('Rows',
+                                      'Columns',
+                                      'SamplesPerPixel',
+                                      'PhotometricInterpretation',
+                                      'BitsAllocated')):
```

Comment:
Preferred formatting

```suggestion
    if all(key in dataset for key in (
        'Rows',
        'Columns',
        'SamplesPerPixel',
        'PhotometricInterpretation',
        'BitsAllocated'
    )):
```